src/common/utxobased/info/dash.ts

```diff
@@ -60,12 +60,12 @@ export const coinInfo: CoinInfo = {
   segwit: false,
   coinType: 5,
   mainnetConstants: {
-    messagePrefix: 'unused',
-    wif: 0xcc,
-    legacyXPriv: 0x02fe52f8,
-    legacyXPub: 0x02fe52cc,
-    pubkeyHash: 0x4c,
-    scriptHash: 0x10
+    messagePrefix: ['unused'],
```

Comment:
If `messagePrefix` doesn't exist for a currency, hardcode to use a default that is bitcoin's messagePrefix.

Reply:
Looks like this prefix is not option according to the `CoinPrefix` type. The reason for this is because AltcoinJS expects `messagePrefix` in its network type it would seem. Making it optional would only mean we need to add more logic to turn it back into an empty/bogus string, so we might as well leave it as a bogus string.